### DISCUSSION TITLE
Artist and Title Order

### DISCUSSION THREAD

**Discussion by alivenine** [Kentakoong/MusicBar#21] [Ideas]
Is there possible to have the option to choose to show the artist first and then the track title or let the user modify the format? Like, "Artist name - Track name", I'm kinda more familiar with this format.

However, it's a great little app!

**Comment by Kentakoong** [comment]
Yep, it would be implemented on the next update! Thank you for the idea

**Comment by alivenine** [comment]
Thanks for considering this! I look forward to the next update~
